mpu6050/registers.go

```diff
@@ -1,132 +1,65 @@
+	return nil
+}
 
-	// Temperature measurement
-	TEMP_OUT_H = 0x41
-	TEMP_OUT_L = 0x42
-
-	// Gyroscope measurements
-	GYRO_XOUT_H = 0x43
-	GYRO_XOUT_L = 0x44
-	GYRO_YOUT_H = 0x45
-	GYRO_YOUT_L = 0x46
-	GYRO_ZOUT_H = 0x47
-	GYRO_ZOUT_L = 0x48
-
-	// External sensor data
-	EXT_SENS_DATA_00 = 0x49
-	EXT_SENS_DATA_01 = 0x4A
-	EXT_SENS_DATA_02 = 0x4B
-	EXT_SENS_DATA_03 = 0x4C
-	EXT_SENS_DATA_04 = 0x4D
-	EXT_SENS_DATA_05 = 0x4E
-	EXT_SENS_DATA_06 = 0x4F
-	EXT_SENS_DATA_07 = 0x50
-	EXT_SENS_DATA_08 = 0x51
-	EXT_SENS_DATA_09 = 0x52
-	EXT_SENS_DATA_10 = 0x53
-	EXT_SENS_DATA_11 = 0x54
-	EXT_SENS_DATA_12 = 0x55
-	EXT_SENS_DATA_13 = 0x56
-	EXT_SENS_DATA_14 = 0x57
-	EXT_SENS_DATA_15 = 0x58
-	EXT_SENS_DATA_16 = 0x59
-	EXT_SENS_DATA_17 = 0x5A
-	EXT_SENS_DATA_18 = 0x5B
-	EXT_SENS_DATA_19 = 0x5C
-	EXT_SENS_DATA_20 = 0x5D
-	EXT_SENS_DATA_21 = 0x5E
-	EXT_SENS_DATA_22 = 0x5F
-	EXT_SENS_DATA_23 = 0x60
-
-	// I2C peripheral data out
-	I2C_PER0_DO      = 0x63
-	I2C_PER1_DO      = 0x64
-	I2C_PER2_DO      = 0x65
-	I2C_PER3_DO      = 0x66
-	I2C_MST_DELAY_CT = 0x67
+// MPU 6050 REGISTER ADRESSES
+const (
+	SMPRT_DIV    uint8 = 0x19
+	CONFIG       uint8 = 0x1A
+	GYRO_CONFIG  uint8 = 0x1B
+	ACCEL_CONFIG uint8 = 0x1C
+	FIFO_EN      uint8 = 0x23
+	I2C_MST_CTRL uint8 = 0x24
+	PWR_MGMT_1   uint8 = 0x6B
+	ACCEL_XOUT_H uint8 = 0x3B
+	TEMP_OUT_H   uint8 = 0x41
+	GYRO_XOUT_H  uint8 = 0x43
+)
 
-	// Clock settings
-	CLOCK_INTERNAL               = 0x00
-	CLOCK_PLL_XGYRO              = 0x01
-	CLOCK_PLL_YGYRO              = 0x02
-	CLOCK_PLL_ZGYRO              = 0x03
-	CLOCK_PLL_EXTERNAL_32_768_KZ = 0x04
-	CLOCK_PLL_EXTERNAL_19_2_MHZ  = 0x05
-	CLOCK_RESERVED               = 0x06
-	CLOCK_STOP                   = 0x07
+// MPU 6050 MASKS
+const (
+	G_FS_SEL     uint8 = 0x18
+	AFS_SEL      uint8 = 0x18
+	CLK_SEL_MASK uint8 = 0x07
+	SLEEP_MASK   uint8 = 0x40
+)
 
-	// Accelerometer settings
-	AFS_RANGE_2G  = 0x00
-	AFS_RANGE_4G  = 0x01
-	AFS_RANGE_8G  = 0x02
-	AFS_RANGE_16G = 0x03
+// MPU 6050 SHIFTS
+const (
+	AFS_SHIFT   uint8 = 3
+	G_FS_SHIFT  uint8 = 3
+	SLEEP_SHIFT uint8 = 6
+)
 
-	// Gyroscope settings
-	FS_RANGE_250  = 0x00
-	FS_RANGE_500  = 0x01
-	FS_RANGE_1000 = 0x02
-	FS_RANGE_2000 = 0x03
+// Gyroscope ranges for Init configuration
+const (
+	GYRO_RANGE_250  byte = 0x00
+	GYRO_RANGE_500  byte = 0x01
+	GYRO_RANGE_1000 byte = 0x02
+	GYRO_RANGE_2000 byte = 0x03
+)
 
-	// other registers
-	SIGNAL_PATH_RES = 0x68 // Signal path reset
-	USER_CTRL       = 0x6A // User control
-	PWR_MGMT_1      = 0x6B // Power Management 1
-	PWR_MGMT_2      = 0x6C // Power Management 2
-	FIFO_COUNTH     = 0x72 // FIFO count registers (high bits)
-	FIFO_COUNTL     = 0x73 // FIFO count registers (low bits)
-	FIFO_R_W        = 0x74 // FIFO read/write
-	WHO_AM_I        = 0x75 // Who am I
+// Accelerometer ranges for Init configuration
+const (
+	ACCEL_RANGE_2  byte = 0x00
+	ACCEL_RANGE_4  byte = 0x01
+	ACCEL_RANGE_8  byte = 0x02
+	ACCEL_RANGE_16 byte = 0x03
```

Comment:
I think it would be nice to be able to share these values across drivers, so that you could use `SetAccelRange` in an interface for any IMU. (This is a more extensive change, so perhaps it's easier to leave this to a future PR).